### DISCUSSION TITLE
Github

### DISCUSSION THREAD

**Discussion by jsmorabito** [athensresearch/athens#1280] [Operations]
How can we set up our Github organization, repositories, issues/labels, permissions, etc to allow for our community to scale?

### current state
- [milestones](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones) (not currently used)
- issue triaging (process formalization in progress #1284)
- issue labels #1055
- epic issues (potentially underutilized. examples #1122 #840 #1119)
- project boards
- teams
- [organization and repository permission levels](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization) not explicit or formalized 

### questions
- what should [github user permissions](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization) be set to for optimal efficiency

### notes
- Preliminary thoughts on scaling github here: https://roamresearch.com/#/app/athensresearch/page/bto7svyxW
- https://blog.zenhub.com/how-to-use-epics-and-milestones/
  - we should use milestones for sprints (ie if we operate in 2 week sprints each sprint is a milestone with issues and objectives located within
  - epics for larger initiatives that collect smaller issues, typically include a mot
    - example epics (website redesign, handbook overhaul, database management, onboarding tour)

**Comment by jefftangx** [comment]
4 larger abstractions for issues:

1. milestones - haven't used yet
1. [epics](https://github.com/athensresearch/athens/labels/epic) (meta-issues) - have made a few. Haven't done a great job of keeping them up to date
1. project boards - have made a few. Haven't done a great job of keeping them up to date
1. Monthly Priorities - past two months #1099  #1271 

Biggest problem with all these is keeping them up to date. Not sure if any/all of them are that good. TODO: see how larger projects do project mgmt and organize larger numbers of issues.
